.circleci/config.yml

```diff
@@ -613,37 +653,45 @@ commands:
 #---------- JOBS ----------
 #--------------------------
 jobs:
-  prepare-and-assemble:
+  assemble:
     executor: ndk-r22-latest-executor
     steps:
       - checkout
-      - when:
-          condition:
-            not:
-              matches:
-                pattern: "^add-changelog.*"
-                value: << pipeline.git.branch >>
-          steps:
-            - restore-gradle-cache
-            - assemble-core-debug
-            - assemble-ui-debug
-            - assemble-core-release
-            - assemble-ui-release
-            - assemble-module:
-                module_target: "instrumentation-tests"
-                variant: "Debug"
-                inject_token: true
-            - assemble-instrumentation-test:
-                module_target: "instrumentation-tests"
-            - assemble-module:
-                module_target: "app-tests-wrapper"
-                variant: "Debug"
-            - assemble-instrumentation-test:
-                module_target: "libnavigation-core"
-                inject_token: true
-            - assemble-instrumentation-test:
-                module_target: "libnavigator"
-            - write-workspace
+      - restore-gradle-cache
+      - assemble-core-debug
+      - assemble-ui-debug
+      - assemble-core-release
+      - assemble-ui-release
+
+  assemble-instrumentation-test:
+    executor: ndk-r22-latest-executor
+    resource_class: medium+
+    steps:
+      - checkout
+      - restore-gradle-cache
+      - assemble-module:
+          module_target: "instrumentation-tests"
+          variant: "Debug"
```

Comment:
Have we always assembled a debug apk?

Reply:
It was in the original config. Do we have differences debug/release variants here?

Reply:
Not really.